cmd/dlt/commands/connections.go

```diff
@@ -101,16 +103,17 @@ var connectionsListCmd = &cobra.Command{
 
 		if len(config.Connections) > 0 {
 			columnOffset := 2
-			maxNameLen, maxSourceLen, maxDestinationLen := len("Name"), len("Source"), len("Destination")
+			maxNameLen, maxSourceLen, maxDestinationLen, maxCronLen := len("Name"), len("Source"), len("Destination"), len("Destination")
```

Comment:
Should the last argument be len("Cron") or something like that?

Reply:
Yes, good catch!